rel-eng/push-packages-to-obs.sh

```diff
@@ -155,7 +155,7 @@ function copy_changed_package()
         # tarball diff or rename not necessarily implies content change!
         star="$sdir/$stem"
         ;;
-      *.obsinfo) break;;
```

Comment:
This was a bug, and stopped the compare at `.obsinfo` files :-)

This is what prevented all packages from being submitted, as discussed on-line.